client/network/src/schema/api.v1.proto

```diff
@@ -56,5 +60,9 @@ message BlockData {
 	// doesn't make in possible to differentiate between a lack of justification and an empty
 	// justification.
 	bool is_empty_justification = 7; // optional, false if absent
+	// Justifications if requested.
+	// Unlike the case for single Justifications this will not be empty even if the contained
+	// Justifications itself are present but empty, since they contain the conensus engine ID.
```

Comment:
I don't really understand this comment.

Reply:
Had another go at reformulating it to try to make it a bit more readable :)